src/intercept.c

```diff
@@ -527,7 +530,11 @@ xabort_errno(int error_code, const char *msg)
 	}
 
 	syscall_no_intercept(SYS_write, 2, main_msg, sizeof(main_msg) - 1);
-	syscall_no_intercept(SYS_exit_group, 1);
+
+	/* abort(), but without using glibc */
+	tid = syscall_no_intercept(SYS_gettid);
+	pid = syscall_no_intercept(SYS_getpid);
+	syscall(SYS_tgkill, pid, tid, SIGABRT);
```

Comment:
So, why is it `syscall` here, and why not `syscall_no_intercept`?
Your comment says "without glibc", but `syscall` is a function in `libc`, which is going to be intercepted actually...